Update web reports (#273)
DevExpress/dotnet-eud: eud-for-devexpress-reports/reporting-for-web/report-designer/use-report-elements/manipulate-report-elements/move-and-resize-report-elements.md

![Web Report Designer - Move Report Controls](../../../images/img119278.png)

You can also [select multiple controls](select-report-elements-and-access-their-settings.md) and move them in the same way as individual report controls.

![Web Report Designer - Move Multiple Report Controls](../../../images/eurd-web-move-multiple-report-controls.png)

You can also use the [Report Explorer](../../report-designer-tools/ui-panels/report-explorer.md) to move a control to other bands (except **Detail Report Band**), or into a **Panel** or **Table Cell** control. Select a control and drag it within the Report Explorer. The drop targets are highlighted when you drag the control over them.

![Web Report Designer - Move Label Using Report Explorer](../../../images/eurd-web-report-explorer-reorder-controls.gif)

You can also use the Report Explorer to move table cells inside a table:

![Web Report Designer - Move Table Cell Using Report Explorer](../../../images/web-report-designer-move-table-cells-in-report-explorer.gif)

> [!NOTE]
> You can drag the **Table Of Contents** only to the **Report Header Band** and **Report Footer Band**.

## Resize Report Elements

> Select a control and then drag a rectangle drawn on its edge or corner to resize it.

Reviewer: What do you mean by "a rectangle drawn on its edge"? Does this appear after you hover over the edge or corner?

Author: The rectangle in question is blue, it appears when you try to resize a control (when you drag it by its sides):

<img width="320" height="149" alt="Image" src="https://github.com/user-attachments/assets/fa95488c-861d-4e92-af53-c440313c12ba" />

Reviewer: I see now. And I assume that "it" refers to the control? If so, can we say:

`Select a control and then drag a rectangle that appears on its edge or corner to resize the control.`

Author: ```suggestion
Select a control and then drag a rectangle that appears on its edge or corner to resize the control.
```
